Topic Wise Questions with Solutions/Array/intersection_sortedarray.py

```diff
@@ -0,0 +1,48 @@
+'''
+Program to find intersection of two sorted arrays
+
+Time complexity : 0(m+n)
+
+Sample Test Case:
+Enter the number of elements of array 1 : 2
+Enter the elements of array 1 :
+1
+2
+Enter the number of elements of array 2 : 3
+Enter the elements of array 2 :
+1
+3
+4
+Intersection of the two arrays is :
+1
```

Comment:
Add the sample output and complexity after the code

Reply:
I made the necessary changes by adding the complexity and sample output at the end.